src/BarrelCalorimeterSciGlass_geo.cpp

```diff
@@ -410,4 +410,4 @@ static Ref_t create_detector(Detector &lcdd, xml_h handle,
   return det_element;
 }
 
-DECLARE_DETELEMENT(epic_SciGlassCalorimeter, create_detector)
+DECLARE_DETELEMENT(epic_EcalBarrelSciGlass, create_detector)
```

Comment:
We've been trying, to first order, to keep the name of the source file the same as the name of the element. Would we be able to use EcalBarrelSciGlass_geo.C?

Reply:
Yes, I saw the inconsistency, but the naming convention for other files is not like that yet, so I opted to conform to that first.

Reply:
Can you approve please.

Reply:
We should keep the name of the plugin source file consistent with the definition. Here you are moving from consistent to inconsistent names.

Reply:
Just to be clear, I like the rename of the plugin type and DECLARE_DETELEMENT, as it is clearer, but just would prefer the name of the src file to be `EcalBarrelSciGlass_geo.cpp` to be consistent with the others.

Reply:
No, we shouldn't. The names of the files are not part of the visible API. Renaming all calorimeters is beyond the scope of this PR.

Reply:
> Just to be clear, I like the rename of the plugin type and DECLARE_DETELEMENT, as it is clearer, but just would prefer the name of the src file to be `EcalBarrelSciGlass_geo.cpp` to be consistent with the others.

It's pretty consistent with others:
https://github.com/eic/epic/blob/15b2e4485ec8eafee6084889ca1d6eac20f47937/src/BarrelCalorimeterInterlayers_geo.cpp#L452

Reply:
Ok, let's make them consistent in a future PR then.